ThirdPartyAdapters/applovin/applovin/src/main/java/com/applovin/mediation/AppLovinNativeAdListener.java

```diff
@@ -0,0 +1,107 @@
+package com.applovin.mediation;
+
+import android.content.Context;
+import android.util.Log;
+
+import com.applovin.nativeAds.AppLovinNativeAd;
+import com.applovin.nativeAds.AppLovinNativeAdLoadListener;
+import com.applovin.nativeAds.AppLovinNativeAdPrecacheListener;
+import com.applovin.sdk.AppLovinSdk;
+import com.applovin.sdk.AppLovinSdkUtils;
+import com.google.android.gms.ads.AdRequest;
+import com.google.android.gms.ads.mediation.MediationNativeListener;
+
+import java.lang.ref.WeakReference;
+import java.util.List;
+
+public class AppLovinNativeAdListener
+        implements AppLovinNativeAdLoadListener, AppLovinNativeAdPrecacheListener {
+
+    private AppLovinNativeAdapter adapter;
```

Comment:
Lets follow Android/oldschool Java/ (and now AppLovin) conventions of prefixing fields with `mXXX`. So this would be `mAdapter`. I don't love it either, but I see that's how Android design theri classes.

Reply:
You can also mark all of these as `final`, since they are assigned in the constructor.